openlibrary/core/schema.py

```diff
@@ -39,6 +39,53 @@ def get_schema():
 
     CREATE INDEX thing_olid_idx ON thing(get_olid(key));
 
+    CREATE TABLE likes (
```

Comment:
we may be able to deprecate this, though might make this point a separate PR (remains to be decided exactly how we're going to handle liking + voting + rating, etc (agreed it shouldn't have been on this branch) :)

Reply:
My guess would be dropping the tables on prod would pretty easy? I think it would be nice to drop both these tables since they're not used now, and they might even change when the full ratings/likes/? features get pushed. But your call.